packages/docs/docs/guides/5-request-client/1-payment-networks.md

```diff
@@ -5,55 +5,86 @@ keywords: [Request Client, Payment networks]
 
 ---
 
-## Other currencies 
-_VRO: this should be modified/added at the end of this section_
+When a user creates and sends a request, he expects to receive the correct amount of money. But how does he keep track of the payments due and received? Request is a ledger that documents requests for payment but doesn't directly interact with any external blockchain where the payment happens. 
 
-If you don't see a currency you would like to create request with, there is two options:
-- Create a declarative request (see [declarative request](TODO))
-- Contact us for your currency requirements: **Join the Request Hub** [**here**](https://join.slack.com/t/requesthub/shared_invite/enQtMjkwNDQwMzUwMjI3LWNlYTlmODViMmE3MzY0MWFiMTUzYmNiMWEyZmNiNWZhMjM3MTEzN2JkZTMxN2FhN2NmODFkNmU5MDBmOTUwMjA)
-- Contribute to the protocol creating a dedicated payment network for this currency, by:
-  - Writing the specification (following the [advanced logic specification](https://github.com/RequestNetwork/requestNetwork/blob/master/packages/advanced-logic/specs/advanced-logic-specs-0.1.0.md) and get inspired by the [others payment networks](https://github.com/RequestNetwork/requestNetwork/tree/master/packages/advanced-logic/specs))
-  - Developing the new payment network in the [advanced-logic package](https://github.com/RequestNetwork/requestNetwork/tree/master/packages/advanced-logic/src/extensions/payment-network).
-  - Developing the payment detection in the payment [payment-detection package](https://github.com/RequestNetwork/requestNetwork/tree/master/packages/payment-detection).
-  - (OPTIONAL) Developing the payment processing in the [payment-processor package](https://github.com/RequestNetwork/requestNetwork/tree/master/packages/payment-detection)
+There are different methods available for the payee and payer to agree on the payment status, and that is when payment networks come into play. **A payment network is a predefined set of rules on how to agree on the payment status of a specific request.**
+
+A payment network is defined by:
+* The information, defined at the request creation, necessary to be able to detect payments
+* A payment method, the method used to perform a detectable payment
+* A payment detection method, the process to determine the amount paid by the payer through the payment method
+
+## Types of payment network
+
+There are currently three types of payment network.
+
+### Address based
+
+For this payment network, a request contains one payment address and optionally one refund address.
```

Comment:
I think we should't mention the refund address mixed with the general payment address.
It breaks the flow and makes things a bit more complex to read.
What if we only talked about refund at a line in the end? 
Like:
`Similar to the payment address, in this payment network we can set a refund address that follows the same rules (it also need to be exclusive to the request).`

Reply:
And of course, something similar for the other payment networks.

Reply:
Good comment, I didn't change for the Declarative description, I think it's better to mix both to better understand this payment network